mockredis/script.py

```diff
@@ -2,8 +2,26 @@
 import threading
 from mockredis.exceptions import ResponseError
 
+
 LuaLock = threading.Lock()
 
+if sys.version_info[0] == 3:
```

Comment:
Python 2 should be the special case here. In future, when Python 4 gets released, this `if` will fallback to the logics for Python 2, which will definitely be incompatible with Python 4.